src/sentry/autofix/utils.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+import requests
+from django.conf import settings
+
+from sentry.utils import json
+
+
+def get_autofix_state_from_pr_id(provider: str, pr_id: int) -> dict[str, Any] | None:
+    response = requests.post(
+        f"{settings.SEER_AUTOFIX_URL}/v1/automation/autofix/get-state-from-pr",
+        data=json.dumps(
+            {
+                "provider": provider,
+                "pr_id": pr_id,
+            }
+        ),
+        headers={"content-type": "application/json;charset=utf-8"},
+    )
+
+    response.raise_for_status()
```

Comment:
will errors be raised from the seer url? `raise_for_status` doesn't raise errors with a helpful error message, this is the code

```
        if 400 <= self.status_code < 500:
            http_error_msg = (
                f"{self.status_code} Client Error: {reason} for url: {self.url}"
            )

        elif 500 <= self.status_code < 600:
            http_error_msg = (
                f"{self.status_code} Server Error: {reason} for url: {self.url}"
            )

        if http_error_msg:
            raise HTTPError(http_error_msg, response=self)
```

Reply:
Shouldn't raise any predefined errors, and any undefined error we capture to sentry on the seer side and we can handle it from there